pkgs/development/python-modules/rtp/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3
+
+# buildInputs
+, mypy
+, hypothesis
+
+# propagatedBuildInputs
+, six
+, flask
+}:
+
+buildPythonPackage rec {
+  pname = "rtp";
+  version = "0.0.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-xUfI8lQD/EhA4NrE4y+KHlie+wwBWkYkDqFLVWKwOzA=";
+  };
+
+  buildInputs = [
+    mypy
+    hypothesis
+  ];
```

```suggestion
  nativeCheckInputs = [
    hypothesis
    unittestCheckHook
  ];
```
We don't want to run mypy tests because upstream is responsible for making those work. We only need to make sure the package works correctly with the dependencies we have packaged.

---

Roget that. Removed mypy from the dependencies with sed, to build the package without it.